WalletWasabi.Backend/wwwroot/index.html

```diff
@@ -12,649 +13,278 @@
-                                    <path d="M23.387 32.258H25v-1.613h.806a3.229 3.229 0 0 0 3.226-3.226c0-.962-.423-1.828-1.094-2.419a3.218 3.218 0 0 0 1.094-2.42 3.229 3.229 0 0 0-3.226-3.225H25v-1.613h-1.613v1.613h-2.42v11.29h2.42v1.613zm-.806-11.29h3.225c.89 0 1.613.723 1.613 1.613s-.723 1.613-1.613 1.613h-3.225v-3.226zm0 4.838h3.225c.89 0 1.613.724 1.613 1.613 0 .89-.723 1.613-1.613 1.613h-3.225v-3.226zM11.29 9.677h1.613v1.613H11.29zM11.29 36.29h1.613v1.613H11.29zM8.065 24.194h1.612v1.612H8.065zM37.097 9.677h1.613v1.613h-1.613zM37.097 36.29h1.613v1.613h-1.613zM40.323 24.194h1.612v1.612h-1.612zM1.613 45.161h1.613v1.613H1.613zM0 46.774h1.613v1.613H0zM1.613 48.387h1.613V50H1.613zM3.226 46.774h1.613v1.613H3.226zM9.677 45.968h1.613v1.613H9.677zM3.226 38.71h1.613v1.613H3.226zM46.774 45.161h1.613v1.613h-1.613zM48.387 46.774H50v1.613h-1.613zM46.774 48.387h1.613V50h-1.613zM45.161 46.774h1.613v1.613h-1.613zM38.71 45.968h1.613v1.613H38.71zM45.161 38.71h1.613v1.613h-1.613zM1.613 3.226h1.613v1.613H1.613zM0 1.613h1.613v1.613H0zM1.613 0h1.613v1.613H1.613zM3.226 1.613h1.613v1.613H3.226zM3.226 9.677h1.613v1.613H3.226zM46.774 3.226h1.613v1.613h-1.613zM48.387 1.613H50v1.613h-1.613zM46.774 0h1.613v1.613h-1.613zM45.161 1.613h1.613v1.613h-1.613zM45.161 9.677h1.613v1.613h-1.613z" />
-                                    <path d="M30.724 31.771a8.872 8.872 0 0 1-5.724 2.1c-4.891 0-8.871-3.98-8.871-8.871 0-2.092.746-4.125 2.1-5.725l-1.23-1.042c-.086.102-.162.21-.243.315h-3.853v-5.645H11.29v7.258h4.425a10.49 10.49 0 0 0-1.163 4.033H11.29v1.612h3.267a10.43 10.43 0 0 0 .768 3.226H11.29v5.645h1.613v-4.032h3.277a10.48 10.48 0 0 0 8.014 4.798v5.686h1.612v-5.68a10.493 10.493 0 0 0 5.961-2.447l-1.043-1.231z" />
-                                </g>
-                            </svg>
-                            <br />
-                            <span class="text-uppercase font-weight-bold">Coin-Control Features</span>
-                        </a>
+            </div>
+        </section>
+        <section class="home p-100">
+            <div class="container">
+                <div class="row">
+                    <div class="col-xl-10 offset-xl-1">
+                        <img src="img/screenshots/MainPage_montage.png" class="w-100" />
```

Comment:
Maybe here is a good place to highlight how easy it is to use Bitcoin privately with Wasabi. Something like this:

```
## Using Bitcoin privately doesn't have to be complicated. Wasabi makes it effortless.

1. Receive some bitcoin to an address.
2. Wait, and let Wasabi do it's magic.
3. Spend it, and rest asured that your privacy is protected.
```

This should be in three columns, with screenshots of the appropriate UI parts.